mindmaps-graql/src/main/java/io/mindmaps/graql/internal/query/ComputeQueryImpl.java

```diff
@@ -68,7 +65,11 @@ public Object execute() throws ExecutionException, InterruptedException {
                 return analytics.degrees();
             }
             case "degreesAndPersist": {
```

Comment:
Hard coded strings ? Can we not add this to one our many enums ? Hard coded strings make me cry.